### PR TITLE
Added Navbar

### LINKED ISSUE #2: Navbar Added
Nav bar is added that will be common for all pages

### PR DESCRIPTION
**Pull request by Raghav888**
Nav bar is added that will be common for all pages

### CODE REVIEW

File: src/components/shared/navbar/Navbar.js [hunk 1]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./Navbar.css"
+export function Navbar(){
+    return(
+        
+    <nav className="mantra-nav">
+    <div className="mantra-nav-title">
+        <a className="mantra-title" href="index.html"> BookStore </a>
```
Comment (tusharkandpal) [inline src/components/shared/navbar/Navbar.js:8]: You can remove all `a` tags in this component.

**Review by tusharkandpal — CHANGES_REQUESTED** [review]
Minor suggestion below else is good.

File: src/components/shared/navbar/Navbar.js (hunk 1 above)
Comment (Raghav888) [reply src/components/shared/navbar/Navbar.js:8]: I will remove it, once I create all other components

Comment (tusharkandpal) [reply src/components/shared/navbar/Navbar.js:8]: Alright.

**Review by tusharkandpal — APPROVED** [review]
You can go ahead & merge.

File: src/components/shared/navbar/Navbar.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
+import React from "react";
+import "./Navbar.css"
+export function Navbar(){
+    return(
```
Comment (mariyasada) [inline src/components/shared/navbar/Navbar.js:4]: use es6 format

File: src/pages/landing/Home.js [hunk 3]
```diff
@@ -0,0 +1,9 @@
+import React from "react";
+import { Navbar } from "../../components/shared/navbar/Navbar";
+export function  Home(){
+    return(
```
Comment (mariyasada) [inline src/pages/landing/Home.js:4]: same here

**Review by mariyasada — APPROVED** [review]

File: src/components/shared/navbar/Navbar.js (hunk 2 above)
Comment (Raghav888) [reply src/components/shared/navbar/Navbar.js:4]: Will do it

File: src/pages/landing/Home.js (hunk 3 above)
Comment (Raghav888) [reply src/pages/landing/Home.js:4]: Will do it

**Pull request merged**